app.js

```diff
@@ -17,7 +13,7 @@ var config = {
 };
 
 
-require(path.join(__dirname, "index.js")).connect(function (context) {
+require(path.join('./index.js')).connect(function (context) {
```

Comment:
the `__dirname` stuff is required here, otherwise you get `Error: Cannot find module 'index.js'`